pkg/device/probe_linux.go

```diff
@@ -52,31 +53,31 @@ func newDevice(
 	}
 
 	if device.Size, err = getSize(name); err != nil {
-		return nil, err
+		return nil, fmt.Errorf("unable to get size for device %v; %v", name, err)
```

Comment:
1. Use `%w` for `err` 
2. you could format it like `unable to get size; device=%v; err=%w`